hack/cluster-push.sh

```diff
@@ -29,7 +29,7 @@ fi
 builder_secretid=$(oc get -n openshift-machine-config-operator secret | egrep '^builder-token-'| head -1 | cut -f 1 -d ' ')
 secret="$(oc get -n openshift-machine-config-operator -o json secret/${builder_secretid} | jq -r '.data.token' | base64 -d)"
 
-if [[ "${podman}" =~ "docker" ]]; then
+if [[ "$@{podman}" =~ "docker" ]]; then
```

Comment:
Huh...what is the `$@` here?  I'd have written `${podman:-}` which is the incantation for "expand to nothing if unset".  Playing around with this:
```
$ echo $@{podman}
{podman}
```

Seems not like what we want?

Reply:
Ah ok, thats what I thought `$@` did clearly I was wrong. Will update.